examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -362,24 +363,70 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
 
           case 'pointmap':
 
-            var pointsJson = (JSON.stringify({
-              dataset: parameters.dataset,
-              filter: getFilter(parameters, defaultNonSamplingDayRange, parameters.geoIds),
-              select: {
-                order: ["-create_at"],
-                limit: defaultPointmapLimit,
-                offset: 0,
-                field: ["id", "coordinate", "place.bounding_box", "create_at", "user.id"]
+            function byPointsTimeRequest(parameters) {
+              return {
+                dataset: parameters.dataset,
+                filter: getFilter(parameters, defaultPointmapSamplingDayRange, parameters.geoIds),
+                group: {
+                  by: [{
+                    field: "create_at",
+                    apply: {
+                      name: "interval",
+                      args: {
+                        unit: parameters.timeBin
+                      }
+                    },
+                    as: parameters.timeBin
+                  }],
+                  aggregate: [{
+                    field: "*",
+                    apply: {
+                      name: "count"
+                    },
+                    as: "count"
+                  }]
+                }
+              };
+            }
+
+            function byPointsRequest(parameters) {
+              return {
+                dataset: parameters.dataset,
+                filter: getFilter(parameters, defaultPointmapSamplingDayRange, parameters.geoIds),
+                select: {
+                  order: ["-create_at"],
+                  limit: defaultPointmapLimit,
+                  offset: 0,
+                  field: ["id", "coordinate", "place.bounding_box", "create_at", "user.id"]
+                },
+                transform: {
+                  wrap: {
+                    key: "points"
+                  }
+                }
+              };
+            }
+
+            var batchPointMapRequest = cloudberryConfig.querySliceMills > 0 ? (JSON.stringify({
```

Comment:
does this request work? @haochen07 , we don't allow the limit inside a `batch` request, right?